tests/Doctrine/Tests/Models/OneToOneSingleTableInheritance/Cat.php

```diff
@@ -8,8 +8,8 @@
 class Cat extends Pet
 {
     /**
-     * @OneToOne(targetEntity="LitterBox")
      * @var LitterBox
+     * @OneToOne(targetEntity="LitterBox")
      */
     public $litterBox;
```

Comment:
Now that's an interesting code example for a test :cat: :cat2:

Reply:
Didn't even notice :joy_cat: